### ISSUE TITLE
Difference between Silva Junior et al 2020 and Mapbiomas Secondary vegetation age

### ISSUE THREAD

**Issue by jecogeo** [celsohlsj/gee_brazil_sv#1]
Hi Celso, I'm diving into your paper and the ATBD of Mapbiomas V6 trying to understand the differences between methods. I'm used to using secondary vegetation age from mapbiomas collection 6 (projects/mapbiomas-workspace/public/collection6/mapbiomas_collection60_secondary_vegetation_age_v1). Now I came through your paper and I'm trying to understand the differences. As far as I understood, you use Mapbiomas v4.1 and your maps (secondary vegetation age) cover the period 1986 - 2018, while the secondary vegetation age from Mapbiomas V6 cover the period 1986-2019. Also, Mapbiomas seems to be more conservative, as it considers pixels detected in 3 consecutive years to calculate ages, else it's not computed.

What would you consider as key points of difference comparing your product with mapbiomas (secondary vegetation age)?
Thanks

**Comment by celsohlsj** [comment]
Hi Jefferson,

Our SF map only considers forest class at annual scale; Also, we only
consider growth when the pixels had a previous anthropic state (eg,
pasture). MapBiomas does not calculate age and considers secondary growth
of forest and non-forest formations.

Best,
Celso.

Em ter., 1 de fev. de 2022 12:02, Jefferson Ferreira-Ferreira <
***@***.***> escreveu:

**Comment by jecogeo** [comment]
> Hi Jefferson, Our SF map only considers forest class at annual scale; Also, we only consider growth when the pixels had a previous anthropic state (eg, pasture). MapBiomas does not calculate age and considers secondary growth of forest and non-forest formations. Best, Celso. Em ter., 1 de fev. de 2022 12:02, Jefferson Ferreira-Ferreira < ***@***.***> escreveu:

Hey Celso, thanks for your reply. I got your points. However, mapbiomas **does provide secondary vegetation ages**, since collection 5. It is considered vegetation growth when the pixels had previous anthropic state (just like yours) and besides it is only considered regrowth if a pixel is detected as secondary vegetation for 3 consecutive years. Then you'll have the ages. The pixels are coded as age + LULC class, so that 1003 is secondary vegetation of 10 years old in forests (class 03). 

Take a look at `projects/mapbiomas-workspace/public/collection6/mapbiomas_collection60_secondary_vegetation_age_v1` on GEE, as I already pointed earlier or in my script for Cerrado biome if want to take a look: https://code.earthengine.google.com/ae93558ba75a7ec206f5e026d1e97de8

Given that, I think the differences between your map and Mapbiomas are the conservative approach of mapbiomas and the fact that they provide ages for all LULC classes, while your dataset is only for forests (which could be easily filtered on GEE using `.remap` function on the mapbiomas dataset).
